### DISCUSSION TITLE
Repo Back Button

### DISCUSSION THREAD

**Discussion by rovo79** [community/community#114513] [Mobile]
Would be great if there was a way to hit back to a repo you were previously viewing. On occasion I am dropped into the app from an external source, and it pushes out of the window the repo I had been previously reading, but now can't find. 
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.154.0 (125806213)
Device: iPad (6th generation) (iOS 17.3.1)
</details>

**Comment by elisealix22** [comment]
Hi @rovo79, thank you for the thoughtful suggestion! I opened an issue to discuss this internally. Let us know if you have any more suggestions or feedback in the meantime!

**Reply by rovo79** [reply]
I think I identified part of the issue. On my iPhone, I see Back button, but on my iPad I do not. I tested this arriving at the same Repo the same path.
iPad:
![image](https://github.com/community/community/assets/117691/b71f5364-4b66-483d-96b7-fb75784106bb)

iPhone:
![image](https://github.com/community/community/assets/117691/169ea9c3-a55c-48e9-b95e-e6f8ee3a7801)

**Comment by Olayiwolaaa** [comment]
If this repo was searched, then check the search section, else, i'd advice you temporarily star the repo if it seems interesting.
I dont own an Ipad but it seems like an issue that might be possible

**Reply by rovo79** [reply]
![image](https://github.com/community/community/assets/117691/2b254625-9cc6-4f7d-ac68-c361efe32dab)

Good suggestion. It's possible I make excessive use of "starring" repos already...

**Reply by Olayiwolaaa** [reply]
# Be Sure to mark this as the answer ☺️, for the badge .
